src/mappings/Tranche.ts

```diff
@@ -33,7 +31,7 @@ export function handleSupplyOrder(call: SupplyOrderCall): void {
   tk.owners = pushUnique(tk.owners, account)
   tk.save()
 
-  let tokenBalance = loadOrCreateTokenBalance(account, token)
-  tokenBalance.pendingSupplyCurrency = amount
-  tokenBalance.save()
+  let tb = loadOrCreateTokenBalance(account, token)
+  calculateDisburse(tb, <PoolAddresses>poolAddresses)
```

Comment:
cc @lucasvo can you clarify here if calling calcDisburse is functionally the same as setting the pendingSupplyCurrency to the amount